### DISCUSSION TITLE
Achievement badges

### DISCUSSION THREAD

**Discussion by lodgendury** [community/community#18578] [Other Feature Feedback, Questions, & Ideas]
These achievement badges absolutely rock!

**Comment by number0x0001** [comment]
Cool

**Reply by Tanmay-Tiwaricyber** [reply]
AMAZING

**Reply by VeXHarbinger** [reply]
I guess we do need some stink'n badges

**Reply by astianmuchui** [reply]
They kind of tell us we are doing something

**Reply by sourabmaity** [reply]
Good

**Reply by jycouet** [reply]
Nice feature!
Let's collect all of them 🦄

**Reply by AbhimanyuAjudiya** [reply]
Cool Designs!!!

**Comment by Caypoh23** [comment]
Now try to get em

**Reply by astianmuchui** [reply]
You can try the following , Get two pull requests merged , That will get you a pull shark

**Comment by sonawaneutkarsh** [comment]
yeah

**Comment by Jeevan1310** [comment]
It was really good

**Reply by Tanmay-Tiwaricyber** [reply]
I think so too😍!

For full details on how to avail these badges you can check out this:

https://github.com/Schweinepriester/github-profile-achievements

**Comment by d-coder111** [comment]
> These achievement badges absolutely rock!

I think so too😍!

For full details on how to avail these badges you can check out this:

https://github.com/Schweinepriester/github-profile-achievements

Good luck😊👍!!!

**Comment by R3SHUT3** [comment]
These achievement badges are really cool

**Comment by GibzB** [comment]
Now, get busy earning them.

**Reply by Tanmay-Tiwaricyber** [reply]
These achievement badges absolutely rock!

I think so too😍!

For full details on how to avail these badges you can check out this:

https://github.com/Schweinepriester/github-profile-achievements

Good luck😊👍!!!

**Comment by IUlok** [comment]
yep

**Reply by Tanmay-Tiwaricyber** [reply]
HMM

**Comment by Tanmay-Tiwaricyber** [comment]
These achievement badges absolutely rock!

I think so too😍!

For full details on how to avail these badges you can check out this:

https://github.com/Schweinepriester/github-profile-achievements

Good luck😊👍!!!

**Comment by Tanmay-Tiwaricyber** [comment]
THESE BADGES ARE GOOOD

**Comment by nemchik** [comment]
Very cool! I hope to see more badges added. I've used https://github.com/ryo-ma/github-profile-trophy for some time, and I think adding a few more badges/achievements and tiers would be great!

**Comment by SebastianMajcher2005** [comment]
These badges are cool. From what I know, many people liked them too. 👍

**Comment by sourabmaity** [comment]
These badges are so cool.🧠

**Comment by goyalyashpal** [comment]
yes, it feels good, made me smile 😊😊

**Comment by aasisodiya** [comment]
I'm loving this kind of Gamification

**Comment by unw9527** [comment]
Cool. Lemme get some as well
